examples/bri-3/front-end/src/login/Login.js

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+import { SRI_BACKEND } from '../constants';
+
+const handleLogin = async () => {
+  if (window.ethereum) {
+    window.ethereum.enable();
+    const accounts = await window.ethereum.send('eth_requestAccounts');
```

Comment:
Also got this warning in the browser console: MetaMask: 'ethereum.send(...)' is deprecated and may be removed in the future. Please use 'ethereum.sendAsync(...)' or 'ethereum.request(...)' instead.
For more information, see: https://eips.ethereum.org/EIPS/eip-1193